components/dashboard/src/SwitchToPAYG.tsx

```diff
@@ -429,35 +441,41 @@ function SwitchToPAYG() {
     const planDescription = pageState.old?.planDetails || "";
     const selectorEntries = getOrganizationSelectorEntries(teams || [], setSelectedOrganization);
     return (
-        <div className="flex flex-col max-h-screen max-w-3xl mx-auto items-center w-full mt-24">
-            <h1>{`Update your ${titleModifier}`}</h1>
-            <div className="w-full text-gray-500 text-center">
-                Switch to the new pricing model to keep uninterrupted access and get <strong>large workspaces</strong>{" "}
-                and <strong>custom timeouts</strong>.{" "}
+        <div className="flex flex-col max-h-screen max-w-2xl mx-auto items-center mt-24">
+            <Heading1>{`Update your ${titleModifier}`}</Heading1>
+            <Subheading className="w-full mt-3 text-gray-500 text-center">
+                Switch to the new pricing model to keep uninterrupted access and <br /> get{" "}
+                <strong>large workspaces</strong> and <strong>custom timeouts</strong>.{" "}
                 <a
                     className="gp-link"
                     href="https://www.gitpod.io/blog/introducing-workspace-classes-and-flexible-pricing"
                 >
                     Learn more →
                 </a>
-            </div>
-            <div className="mt-6 space-x-3 flex">
+            </Subheading>
+            <div className="mt-7 space-x-3 flex">
                 {renderCard({
                     headline: "LEGACY PLAN",
                     title: planName,
-                    description: planDescription,
+                    description: <div className="mb-5">{planDescription}</div>,
                     selected: false,
                     action: (
-                        <Alert type="error">
-                            Discontinued on <strong>March 31st</strong>
-                        </Alert>
+                        <div className="flex">
+                            <span className="text-red-600 dark:text-red-200">
+                                Discontinued on <strong>March 31st</strong>
+                            </span>
+                        </div>
                     ),
                     additionalStyles: "",
                 })}
                 {renderCard({
                     headline: "NEW PLAN",
                     title: "$9 / month (1,000 credits)",
-                    description: "Pay-as-you-go after that for $0.036 per credit.",
+                    description: (
+                        <>
+                            Pay-as-you-go after that for <br /> $0.036 per credit.
+                        </>
+                    ),
                     selected: true,
                     action: (
```

Comment:
**question**: Since we're not making this as a forced flow for users, what do you think it's worth adding a button to skip for now? Thinking that there's no other clear indication, you can bypass this or a cancel button. Button label could be _Skip for now_ or _Reminder me later_. Cc @geropl @atduarte 

| BEFORE | AFTER |
|-|-|
| <img width="393" alt="actions-before" src="https://user-images.githubusercontent.com/120486/223406552-e64edabe-bf33-4354-b856-5d4c44b1ca7c.png"> | <img width="393" alt="actions-after" src="https://user-images.githubusercontent.com/120486/223406556-306b6976-c258-40c0-8319-b36d2469147b.png"> |